doc/cluster-management/olvm.md

```diff
@@ -0,0 +1,546 @@
+**NOTE: This is a developer release**
+
+# OLVM Cluster API Provider
+The OLVM Cluster API Provider allows you to create Kubernetes clusters on an
```

Comment:
The first instance of an acronym should be spelled out.  "Oracle Linux Virtualization Manager (OLVM)".

Reply:
yeah i thought i did that, will fix